Hackney.Core/Hackney.Core.Sns/README.md

```diff
@@ -0,0 +1,55 @@
+# `Hackney.Core.Sns` NuGet Package
+
+#### Sns Gateway
+The `SnsGateway` implementation of the `ISnsGateway` interface allows the easy publishing of an event message to an Sns topic.
+The `ISnsGateway` interface is made available by using the `AddSnsGateway()` extension method during your application start-up.
+
+#### Shared Classes
+- EntityEventSns - Model of the event message received by a function
+- EventData - Contains the data changed in an event
+- User - Contains information about a user triggering an event
+- EventTypes - Names all events we are currently using
+
+#### SnsIntialisationExtensions
+
+The `SnsIntialisationExtensions` class configures an application to the Amazon Simple Notification Service (SNS). This will allow the application to interact with SNS Topics and subscriptions.
+
+#### Usage
+
```

Comment:
You've got usage details for setting up the env var, but not for actually using the extension method.
It should be similar to others like [this one](https://github.com/LBHackney-IT/lbh-core#usage-3).